lesson_7/turcu_alexandru/decorators.py

```diff
@@ -0,0 +1,61 @@
+def uppercase(greetings):
```

Comment:
who's file is this? Is it yours?

Reply:
yes. when we receive multiple problems I always try to solve them step-by-step. after solving one I comment it and start another. I proposed myself to un-comment them at the end but I forgot.

Reply:
I was referring to the fact that the file has the following name: "lesson_7/turcu_alexandru/decorators.py "

Reply:
To be honest I'm not sure how I did that. Yes, your're right, that code is not mine.